provision-nginx.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+# shellcheck disable=1091
+. mail-toaster.sh || exit
+
+# shellcheck disable=2016
+export JAIL_CONF_EXTRA="
+		mount += \"$ZFS_DATA_MNT/webmail \$path/data nullfs rw 0 0\";"
+
```

Comment:
Does nginx need a persistent data directory? If so, then either a) it should be mounted somewhere useful like /usr/local/www/nginx, or b) nginx's config should point to /data/.

Some thought should also be given to how one would manage nginx config & data. Perhaps /data/nginx/etc and /data/nginx/htdocs should be created and configured, or something to that effect.